src/main/java/com/ericsson/ei/controller/model/ParseInstanceInfoEI.java

```diff
@@ -19,6 +19,7 @@
 import com.ericsson.ei.erqueryservice.ERQueryService;
 import com.ericsson.ei.handlers.ObjectHandler;
 import com.ericsson.ei.handlers.RMQHandler;
```

Comment:
I guess we don't need the import for RMQHandler now?

Reply:
Done in upcoming commit